apis/core/v1alpha1/conditions.go

```diff
@@ -23,6 +23,9 @@ import (
 type ConditionType string
 
 const (
+	// ConditionTypeAdopted indicates that the adopted resource custom resource
+	// has been successfully reconciled and the target has been created
+	ConditionTypeAdopted ConditionType = "ACK.Adopted"
```

Comment:
Is this condition to be applied to the `AdoptedResource` CR or to the target resource CR? If the latter, then perhaps we don't need a `services.k8s.aws/adopted-on` annotation at all, since the Condition has a timestamp assocated with its creation.

Reply:
This condition is currently being applied to the `AdoptedResource` CR to describe to the user that it has essentially reached the terminal condition.